src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -64,6 +67,61 @@ public void tearDown() throws InterruptedException {
     service.resetNumOfRetryToTriggerTelemetry();
   }
 
+  /**
+   * Verify the passed heartbeat frequency matches the output value if the input is valid (between
+   * 900 and 3600).
+   */
+  @Test
+  public void testHeartbeatFrequencyValidValue() throws Exception {
+    Properties paramProperties = new Properties();
+    paramProperties.put(CLIENT_SESSION_KEEP_ALIVE_HEARTBEAT_FREQUENCY, 1800);
+    Connection connection = getConnection(paramProperties);
+
+    connection.getClientInfo(CLIENT_SESSION_KEEP_ALIVE_HEARTBEAT_FREQUENCY);
```

Comment:
Why do we need this line?

Reply:
oops, left that in there by accident while debugging! Good catch; I'll delete it.